### ISSUE TITLE
Cannot deploy

### ISSUE THREAD

**Issue by lilyx13** [containers/website-new#13]
I'm not able to deploy the site using `yarn deploy`.=

## Expected behaviour
- run `yarn deploy`
- the automated push will generate the build directory from which the site is hosted

## What happens
```
[INFO] `git commit -m "Deploy website - based on 83ae00881f9cf65bd623bf31b5835bc3fa69d955"` code: 0
ERROR: Permission to containers/containers.github.io.git denied to lilyx13.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
[INFO] `git push --force origin sprint-1` code: 128
[ERROR] Deployment of the build output failed.
[ERROR] Error: Running "git push" command failed. Does the GitHub user account you are using have push access to the repository?
```

## Research
I found this happened to others [this stack overflow post](https://stackoverflow.com/questions/72613299/docusaurus-website-github-pages-deploy-doesnt-work-for-organisations) seems to be the same issue I'm having. It looks like it's solved by either having the right permissions or from the repo being changed from private to public (we are already public).
## Request
**Could someone with higher permissions than me try to run the command and see if it's successful?

**Comment by TomSweeneyRedHat** [comment]
Well, I tried, and failed.  The README.md instructions could use a little more clarity, especially for non-web developers like myself.  I managed to install `nvm` and `npx` and also ran `npx create-docusaurus@latest website-new classic` as suggested by docusaurus.io/docs.  When I tried to start docusaurus with `npx docusaurus start` and fails and log indicates a "can't find it" type of message.

After all of that, I just tried `yarn` as the README.md suggested, but I get a "yarn: command not found" error with that, and figure that out. 

Perhaps add something like:

Fedora Install
```
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
curl -fsSL https://rpm.nodesource.com/setup_19.x | bash -
yum install -y nodejs
npm install --global yarn
```

Now with `yarn`, I get a different error:

```
# yarn
node:internal/modules/cjs/loader:1093
  throw err;
  ^

Error: Cannot find module '/root/tsweeney/workspaces/containers/website-new/.yarn/releases/yarn-3.3.1.cjs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1090:15)
    at Module._load (node:internal/modules/cjs/loader:934:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v19.8.1
```

Let me know if I messed something up, or if others have better luck and more yarn knowledge than I.

**Comment by lilyx13** [comment]
Ah I see. @TomSweeneyRedHat, I think you would want to run `yarn install` from within the repo. then try running `yarn start`.

**Comment by lilyx13** [comment]
I fixed this up. The baseURL needed to be `/websites-new/`, trailingSlash set to `false`, and no `/` at the end of the url. Then I set the deploymentBranch to production and ran just `yarn deploy` and it worked. 

Note that a major determiner in whether or not this worked was to **not** use `GIT_USER=username` along with the `yarn deploy` command. I think that this triggered git to try to use a password instead of ssh or something along those lines which doesn't authenticate right for Github.

Anyway, the `production` branch is where the site is now deployed.
